Makefile

```diff
@@ -70,3 +70,9 @@ beats-dashboards:
 .PHONY: docs
 docs:
 	sh libbeat/scripts/build_docs.sh ${PROJECTS}
+
+.PHONY: package
+package:
+	$(foreach var,$(BEATS),$(MAKE) -C $(var) package || exit 1;)
```

Comment:
I added the parallel builds as a TODO here: https://github.com/elastic/beats/issues/2251